.github/workflows/codeql.yml

```
@@ -13,10 +13,10 @@ name: "CodeQL"
 
 on:
   push:
-    branches: [ main ]
+    branches: [ deepnote ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)